### DISCUSSION TITLE
Deleting Saved Views

### DISCUSSION THREAD

**Discussion by CalderValleyIT** [mattieserver/netbox-topology-views#379] [Q&A] [answered]
Hi,

I feel like I am missing something but for the life of me a cannot seem to find a delete button for saved Views.

I know i'll be kicking myself when someone says 'just click this button!' 

Thank you in advance for your help

**Comment by dreng** [comment, marked as answer]
Hi,

saved filters are a NetBox feature. It is not related to this plugin. Use the menu to expand "Customization" and click on "Saved Filters". You will find the further steps for sure.
